src/main/java/bio/overture/ego/model/dto/ApiKeyScopeResponse.java

```diff
@@ -12,7 +12,6 @@
 @JsonView(Views.REST.class)
 public class ApiKeyScopeResponse {
   private UUID user_id;
-  private String client_id;
```

Comment:
not sure if removing this will break song/score.
since you still have the authorization is there a way to keep this if needed ?

Reply:
Yes, its doable with a bit of moving around where the response object is built and pulling the client ID from the auth.

The `client_id` value is just the application client_id that made the request. No application should need to see this when they generated the original request anyways... so I removed it because it seems to have no value and complicated building the response.

I would prefer removing any dependency on this field vs adding it back in, unless you can see a reason to provide this.

Reply:
yeah I agree that it's better to remove it I'm just wondering if it will break anything, I checked that score is using spring class (RemoteTokensService) which maybe expecting this for some reason, because it seems to be here intentionally, I just don't want to break other services, if you are sure of it no problem from my end

Reply:
Its a valid concern. Hold off on this review until I can confirm what the spring RemoteTokensService expects in the endpoint.